sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/ItemOperations.java

```diff
@@ -31,6 +33,7 @@
     public static <T> FeedResponse<T> readMany(CosmosContainer container,
```

Comment:
These APIs are also in implementation layer, we have ability to still improve them. Shouldn't we change their signature to take `CosmosItemIdentity` ?

Reply:
No - I left this intentionally to not break Adobe. Sounds like we told Adobe to take a dependency on this implementation package class. Whoever worked with Adobe on this needs to follow up with them and ask them to move to the ReadMany Api in the public surface area - only then we will be able to delete this class.